linux/Jenkinsfile

```diff
@@ -1,126 +1,98 @@
-NODE_LABEL = "dockerBuild&&linux&&x64" // Dynamic docker machines from Azure
+/*
+--------------------------------------------
+buildArch   RPM         DEB         Comments
+--------------------------------------------
+x86_64      x86_64      amd64
+armv7hl     armv7hl     armhf       arm32
+aarch64     aarch64     arm64       arm64
+ppc64le     ppc64le     ppc64le
+source      src         -           only for RPM
+s390x       s390x       s390x       only for jdk8+
+*/
+
+env.NODE_LABEL = "dockerBuild&&linux&&x64"
+env.PRODUCT = "temurin"
 
 pipeline {
     agent {
-        label NODE_LABEL
+        label env.NODE_LABEL
```

Comment:
This won't work for debian non linux-x64.

Reply:
good catch! I missed it